wrapper.go

```diff
@@ -13,16 +13,24 @@ var (
 )
 
 type promWrapper struct {
-	namespace string
+	namespace  string
+	collectors []prometheus.Collector
 }
 
 func newPromWrapper(cfg *promConfig) monitor.BricksReporter {
 	return &promWrapper{
-		namespace: cfg.namespace,
+		namespace:  cfg.namespace,
+		collectors: cfg.collectors,
 	}
 }
 
 func (p *promWrapper) Connect(ctx context.Context) error {
+	// Register all predefined Collectors
+	for _, collector := range p.collectors {
+		if err := prometheus.Register(collector); err != nil {
```

Comment:
How do we know that we need to register these collectors and that they were not registered externally? Do we also have to unregister them?

Reply:
Currently we will not allow to "Deregister"